src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/helm.py

```diff
@@ -16,8 +16,13 @@
 logger = get_logger(__name__)
 
 
-# https://github.com/kubernetes/helm/blob/b6660cd5c9a9ee35ff24034fcc223b7eaeed3ee2/pkg/tiller/release_server.go#L80
-VALID_NAME = '^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])+(.tgz)(.prov)?$'
+# Based on the semver regex here:
+# https://github.com/Masterminds/semver/blob/e71d0f94bb19d376417da48392b212ac41be4991/version.go
+VALID_NAME = '^[a-zA-Z0-9]+-'\
```

Comment:
The server side now matches the old regex so this change won't fix the problem until the server side is updated with the new regex. Due to the same region, I think we should remove this client side validation to avoid inconsistencies now and in the future. /cc @yuwaMSFT2 @shizhMSFT

Reply:
I agree. In general, client-side validation in the CLI should be very light, and reserved for instances where inadequacies in the service would result in a very poor or confusing experiences to users. If the server-side error is good, rely on that.